pinot-common/src/main/java/org/apache/pinot/common/request/context/FunctionContext.java

```diff
@@ -84,7 +84,7 @@ public boolean equals(Object o) {
 
   @Override
   public int hashCode() {
-    return 31 * 31 * _type.hashCode() + 31 * _functionName.hashCode() + _arguments.hashCode();
```

Comment:
This change is a bit concerning because it seems like the value of the hashCode must be depended on somehow. If that’s the case, constructing collisions with polynomial hash codes is very easy to do. Do different function contexts need distinct hash codes for this to work properly?

Reply:
I was expected a consistence hash code for ExpressionContext across different JVMs, this is not the case anymore.